eth/ethconfig/config.go

```diff
@@ -66,7 +66,7 @@ var Defaults = Config{
 	RPCGasCap:          50000000,
 	RPCEVMTimeout:      5 * time.Second,
 	GPO:                FullNodeGPO,
-	RPCTxFeeCap:        1, // 1 ether
+	RPCTxFeeCap:        1000, // 1 ether
```

Comment:
The comment should be updated acccordingly.

Reply:
Done in 9174f0f42333c4b52a05e84af1821bf8d32b919e.